esmvalcore/preprocessor/_multimodel.py

```diff
@@ -274,7 +275,13 @@ def _multicube_statistics(cubes, statistics, span):
         raise ValueError('Cannot perform multicube statistics '
                          'for a single cube.')
 
-    copied_cubes = [cube.copy() for cube in cubes]  # avoid modifying inputs
+    copied_cubes = []
+    for cube in cubes:
+        # avoid modifying inputs
+        copied_cube = cube.copy()
+        copied_cube = remove_fx_variables(copied_cube)
```

Comment:
I would suggest to move this line up to line 209 (inside the 'combine' function), because there we already do some other cleanup tasks that need to be done before the cubes can be combined.